metabase/metabase
# 38811 no global css 1

Relates to https://github.com/metabase/metabase/issues/38811 , Milestone 1

### Description

This migrates global CSS files to css modules (but without classes isolation).
+ upgrade css-loader package

### How to verify
- Switch to master branch, build the app with `yarn build`
- Copy `app-main.css`, `styles.css`, `vendor.css` from `resources/frontend_client/app/dist`.
- Switch to this branch, build the app with `yarn build`
- Compare generated `app-main.css`, `styles.css`, `vendor.css` with ones from master branch.

Those should have same content.
